Automattic/jetpack
# Newsletters: update notice to reflect subscribers reached after post was published

Fixes https://github.com/Automattic/gold/issues/46

## Proposed changes:

>  For a future PR, I think it would be nice to adjust NewsletterNotice once a post has been published. We shouldn't display "this post will be sent" for a post that has already been sent to subscribers:

Based on Jeremy's comments in https://github.com/Automattic/jetpack/pull/30668#pullrequestreview-1439305823.

It's important to note that this will need a further iteration as the number of subscribers that are shown in the UI is not static.  There could be a case where a user gained a new subscriber, and the text "This was sent to 30+ subscribers" is shown in the UI when the post was sent to 29+ subscribers. 

I suspect we can set the number of subscribers that were reached in the post meta and return the value from the backend. See follow-up task: https://github.com/Automattic/gold/issues/47.

https://github.com/Automattic/jetpack/assets/7076981/c5843340-bc03-488a-a53f-2242826ff49d. 

### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Create a newsletter post
* Verify the Newsletter Notice is in the future tense ( "This will be sent to" )
*  Publish the post
* Verify the Newsletter Notice is in the past tense ( "This was be sent to" )
